command/expand.go

```diff
@@ -16,22 +16,28 @@ func expandSource(
 	client storage.Storage,
 	followSymlinks bool,
 	srcurl *url.URL,
+	isRaw bool,
 ) (<-chan *storage.Object, error) {
-	var isDir bool
-	// if the source is local, we send a Stat call to know if  we have
-	// directory or file to walk. For remote storage, we don't want to send
-	// Stat since it doesn't have any folder semantics.
-	if !srcurl.HasGlob() && !srcurl.IsRemote() {
-		obj, err := client.Stat(ctx, srcurl)
-		if err != nil {
-			return nil, err
+
+	// If wildcard operations are disabled (raw flag is set), then program deals with 1 file.
+	// Therefore, there is no need to check the directory operations.
```

Comment:
```suggestion
	// if wildcard operations are disabled with the raw flag, then the program deals with 1 file.
	// therefore, there is no need to check if it is a directory.
```
I am not sure about the comment but let's not start with capital letters.

Reply:
fixed the comments as you suggested.